config/development_active.yaml

```diff
@@ -92,6 +92,11 @@ clusterMetadata:
       initialFailoverVersion: 0
       rpcName: "cadence-frontend"
       rpcAddress: "localhost:8933"
+    other:
+      enabled: true
+      initialFailoverVersion: 3
```

Comment:
why not 2? since there is a 0 and 1

Reply:
not suppose to check in this file. Will do it in a separate PR